packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -56,5 +64,62 @@ const SIDEBAR = useSetting("SIDEBAR")
 
 const slots = useSlots()
 
+const props = defineProps({
+  layoutId: {
+    type: String,
+    default: null,
+  },
+})
+
+type PaneEvent = {
+  max: number
+  min: number
+  size: number
+}
+
+const PANE_SIDEBAR_SIZE = ref(25)
+const PANE_MAIN_SIZE = ref(75)
+const PANE_MAIN_TOP_SIZE = ref(45)
+const PANE_MAIN_BOTTOM_SIZE = ref(65)
+const PANE_STORAGE_KEY = `${props.layoutId}-app-pane-layout`
+
+if (!COLUMN_LAYOUT) {
+  PANE_MAIN_TOP_SIZE.value = 50
+  PANE_MAIN_BOTTOM_SIZE.value = 50
+}
+
+function setVerticalPaneEvent(event: PaneEvent[]) {
+  if (!props.layoutId) return
+
+  setLocalConfig(`${PANE_STORAGE_KEY}_vertical`, JSON.stringify(event))
```

Comment:
Don't mix the naming with kebab-case and snake_case. Use either one. IMO the better naming would be like this `${props.layoutId}-pane-config-vr`

Reply:
Here vr and hr could be dynamic if you go with the same method for both.

Reply:
sure